docs/lib/duckdb.md

```diff
@@ -53,7 +53,21 @@ Plot.plot({
 })
 ```
 
-For externally-hosted data, you can create an empty `DuckDBClient` and load a table from a SQL query, say using [`read_parquet`](https://duckdb.org/docs/guides/import/parquet_import) or [`read_csv`](https://duckdb.org/docs/guides/import/csv_import).
+When using file attachments, the following formats are supported for tables: [CSV](./lib/csv), [TSV](./lib/csv), [JSON](./javascript/files#json), [Apache Arrow](./lib/arrow), [Apache Parquet](./lib/arrow#apache-parquet).
+
+You can also [attach](https://duckdb.org/docs/sql/statements/attach) a complete database (saved as DuckDB file, typically using the extension `.db`, `.ddb`, or `.duckdb`):
+
+```js echo
+const db2 = await DuckDBClient.of({base: FileAttachment("quakes.db")});
```

Comment:
If you say this

```suggestion
const db2 = await DuckDBClient.of({main: FileAttachment("quakes.db")});
```

can you then say

```sql
SELECT COUNT() FROM events
```

? I think it should work since`main` is the default schema. Then maybe we could also support this syntax:

```js
const db2 = await DuckDBClient.of(FileAttachment("quakes.db"));
```

Or even:

```js
const db2 = FileAttachment("quakes.db").duckdb();
```

Reply:
I couldn't get `main` to work (but it is doing something, as calling a `main.events` table in that case… errors).

Reply:
Okay, let’s just start here then. 👍